client/src/components/exercise/lab7/Main.js

```diff
@@ -0,0 +1,85 @@
+/* eslint-disable react/prop-types */
+/* eslint-disable require-jsdoc */
```

Comment:
Fill in doc strings on this

Reply:
Resolved. No function other than render.